lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -759,7 +759,8 @@ function dosomething_campaign_add_shipment_form_vars($node) {
  * @return bool
  */
 function dosomething_campaign_is_hot_module($vars) {
-  if (isset($vars['field_high_season'])) {
+  $hot_module_enabled = dosomething_helpers_get_variable('node', $vars['nid'], 'hot_module_enabled');
```

Comment:
I think that seems right, unless I'm missing something... although the function name is weird to me, haha.